tutorials/System.getAllWindows.md

```diff
@@ -0,0 +1,5 @@
+Retrieves an array of data (name, ids, bounds) for all application windows
+# Example
+```js
+fin.System.getAllWindows()then(arr_win => console.log(arr_win)).catch(err => console.log(err));
```

Comment:
This example has a syntax error, missing `.` between `getAllWindows()` and `then`